ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -24,10 +27,17 @@ def Acosh(*, x, name="Acosh"):
     return ivy.acosh(x)
 
 
-Add = tf_frontend.math.add
+Add = to_ivy_arrays_and_back(
+    map_raw_ops_alias(tf_frontend.math.add, kwargs_to_update={"Add": "name"})
```

Comment:
I think the `kwargs_to_update` should be empty for this function as we are not renaming any keyword argument when it gets passed to the actual frontend function from raw_ops, I think given that function checks for the keys being present in both the dictionaries, you may not be able to facing any issues even after adding `{"Add": "name"}`, but I think it should be empty unless the names of the arguments change and "Add" isn't an argument in the funcftion, it's the name of the function instead. Same applies to all the other functions in this file.